src/store/modules/game.js

```diff
@@ -191,7 +191,7 @@ export default {
 		resetPNumIfNull(context) {
 			// Set my pNum if it is null
 			if (context.state.myPNum === null) {
-				let myPNum = context.state.players.findIndex((player) => player.userName === context.rootGetters.myUserName);
+				let myPNum = context.state.players.findIndex((player) => player.username === context.rootGetters.myUserName);
```

Comment:
This reference will change when we deprecate rootGetters.myUsername in favor of `rootState.auth.username;`